service/src/main/java/uk/gov/hmcts/reform/fpl/service/PaymentService.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.fpl.config.LocalAuthorityNameLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.FeesData;
+import uk.gov.hmcts.reform.fpl.model.common.C2DocumentBundle;
+import uk.gov.hmcts.reform.fpl.request.RequestData;
+import uk.gov.hmcts.reform.payment.client.PaymentApi;
+import uk.gov.hmcts.reform.payment.model.CreditAccountPaymentRequest;
+
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.unwrapElements;
+import static uk.gov.hmcts.reform.payment.model.enums.Currency.GBP;
+import static uk.gov.hmcts.reform.payment.model.enums.Service.FPL;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class PaymentService {
+
+    private static final String SITE_ID = "ABA3";
```

Comment:
That could be extracted to the config (maybe even secret?) 🤔 
but I guess just the config.

Reply:
done